### ISSUE TITLE
Can't setup MS Teams alert

### ISSUE THREAD

**Issue by mcg** [getsentry/sentry#20819]
Finishing our setup of the new/official MS Teams Sentry integration. On the Alert screen when I attempt to save the alert, I am receiving this error:

`There was an error saving your changes. Make sure all fields are valid and try again.`

![Sentry 2020-09-16 15-52-17](https://user-images.githubusercontent.com/305/93385882-dc865380-f834-11ea-81cd-77fd1119f782.jpg)

**Comment by scefali** [comment]
@mcg I believe this error has occurred because your Teams administrator has disabled third-party applications. More info here: https://docs.microsoft.com/en-us/microsoftteams/teams-app-permission-policies

**Comment by mcg** [comment]
We had to have the Sentry app allowed for our Teams environment and I was able to see and add the app in Teams after that. I can interact with the bot in Teams and Sentry shows the integration as active. But i cannot add alerts.

**Comment by scefali** [comment]
> We had to have the Sentry app allowed for our Teams environment and I was able to see and add the app in Teams after that. I can interact with the bot in Teams and Sentry shows the integration as active. But i cannot add alerts, due to this error.

I talked to the folks at Microsoft about this. This is what they said:

> That error looks like a Teams admin has disabled third-party applications. If you have visibility into the customer, you could recommend to them that they reach out to their Teams administrator and request the app be enabled for their tenant. Customer implementation thing.

You will need to talk to your admin about this, there's nothing we can do.

**Comment by scefali** [comment]
@mcg 
I talked to the folks at Microsoft about this. This is what they said:

> That error looks like a Teams admin has disabled third-party applications. If you have visibility into the customer, you could recommend to them that they reach out to their Teams administrator and request the app be enabled for their tenant. Customer implementation thing.

You will need to talk to your admin about this, there's nothing we can do.

**Comment by mcg** [comment]
@scefali The app was not available in our Teams. I talked to our admin, they added the app. I then configured the app in our Teams. I received this message in Teams.
![General | Microsoft Teams 2020-09-17 09-23-46](https://user-images.githubusercontent.com/305/93476247-91695080-f8c7-11ea-8277-1330d2ffb355.jpg)

I then headed over to Sentry to try and setup and alert. That is where I had the error. Now this morning, I see that, yesterday, 3 hours after enabling, this message appears in Teams:
![General | Microsoft Teams 2020-09-17 09-26-07](https://user-images.githubusercontent.com/305/93476496-dd1bfa00-f8c7-11ea-99be-248d36d18913.jpg)

Now... finally I can add alerts without an error.

Couple of things... Spamming General in Teams is not great, unsure if that is configurable in the integration, but if possible, that would be good. It may also be helpful to document an expected delay between adding the integration to a team and it actually being enabled/usable, unless this is a bug?

Thanks for the feedback tho and many thanks for the official integration, it helps a lot.
